tests/product/base_product_case.py

```diff
@@ -39,7 +39,7 @@
 
 # TODO: make tests not dependent on the particular version of Presto at
 # http://teradata-download.s3.amazonaws.com/aster/presto/lib/presto-0.101-1.0.x86_64.rpm
-PRESTO_RPM = 'presto-0.101-1.0.x86_64.rpm'
+PRESTO_RPM_GLOB = r'presto-*.x86_64.rpm'
```

Comment:
If we have the rpm filename, then instead of doing a regexp, we should use that.